xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -442,15 +441,16 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem *item, const CGUIInf
       // special handling for channels without epg or with radio rds data
       case PLAYER_TITLE:
       case VIDEOPLAYER_TITLE:
-      {
-        const MUSIC_INFO::CMusicInfoTag* tag =
-            CServiceBroker::GetGUI()->GetInfoManager().GetCurrentSongTag();
-        if (tag && !tag->GetTitle().empty())
         {
-          strValue = tag->GetTitle();
-          return true;
+          if (!epgTag)
+          {
+            if (!item->GetLabel().empty() && item->GetLabel() != channel->ChannelName())
```

Comment:
Sorry,this looks more like a hack and nobody will understand the logic behind. And I'm afraid there is no other logic than "find any condition not that makes the shoutcast use case work."

Reply:
Ok the reason behind the two conditions is there is an user setting that I need to respect to show the 19055 message, and the second is the default label is the Channel.name so it gets repeated if I don't skip it.  I will dig deeper.

Thanks

Martin

Reply:
The latest commit excludes PVR entirely from the PR as the change to Shoutcast message already makes it available in MusicPlayer.Title as it should.  FileItem changed, otherwise the default MusicPlayer.Title is the PVR .pvr filename which does not belong in Music..  Skins can now optionally use MusicPlayer.Title instead of Player.Title to display Shoutcast data.  When there is no Shoutcast data EPG data continues to be displayed using MusicPlayer.Title